src/ui/buttons/props.js

```diff
@@ -696,6 +719,56 @@ export function normalizeButtonStyle(
   };
 }
 
+export function validateButtonMessage(
+  props: ?ButtonPropsInputs,
+  message: ButtonMessageInputs
+): ButtonMessage {
+  if (!message) {
+    throw new Error(`Expected props.message to be set`);
+  }
+
+  props = props || getDefaultButtonPropsInput();
+
+  const { amount, offer, color, position, align } = message;
+
+  if (amount !== undefined) {
+    if (typeof amount !== "number") {
+      throw new TypeError(
+        `Expected style.height to be a number, got: ${amount}`
+      );
+    }
```

Comment:
Can we add an additional check to ensure that the number is positive?

Reply:
will do